tests/dist/python/run_dist_objects.py

```diff
@@ -201,9 +429,11 @@ def test_dist_embedding(g):
     g = dgl.distributed.DistGraph(graph_name, gpb=gpb)
 
     target_func_map = {
+        "DistGraph": test_dist_graph,
+        "DistGraphServices": test_dist_graph_services,
```

Comment:
why create 2 test functions for `DistGraph`? Or could we arrange the tests for `DistGraph` in a more clear way? each pubic method correspond to each test function? or decorated by something like this `@pytest.mark.parametrize("hetero", [False, True])`?

Reply:
one dedicated test for each public method looks more clear. For example, `test_dist_graph_sample_neighbors()`. how do you think about it?

Reply:
In general, if anyone else check the test cases here, he/she could have a clear view of what are tested.